.github/workflows/pr-diff.yml

```
@@ -0,0 +1,82 @@
+name: PR Diff
+
+on:
+  issue_comment:
+    types: [created]
+
+permissions:
+  issues: write
+  pull-requests: write
+  contents: read
+
+jobs:
+  pr_diff:
+    runs-on: ubuntu-latest
+
+    # Only run when a comment containing `/prdiff` is created
+    # and the author is a member, collaborator or owner
+    if: >
+      (
+        github.event_name == 'issue_comment' &&
+        github.event.issue.pull_request &&
+        contains(fromJSON('["MEMBER", "COLLABORATOR", "OWNER"]'), github.event.comment.author_association) &&
+        startsWith(github.event.comment.body, '/prdiff')
+      )
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Read comment
+      id: read_comment
+      env:
+        COMMENT_BODY: ${{ github.event.comment.body }}
+      run: |
+        if [[ "$COMMENT_BODY" =~ ^/prdiff[[:space:]]([^[:space:]]+) ]]; then
+          CMD_ARG=${BASH_REMATCH[1]}
+          echo "Using cmd argument: $CMD_ARG"
+          echo "other_pr=$CMD_ARG" >> $GITHUB_OUTPUT
+        else
+          echo "Comment does not match format: '/prdiff <pr_url>': ignoring"
+        fi
+
+    - name: Get current PR URL
+      if: steps.read_comment.outputs.other_pr
+      id: get_pr_url
+      run: |
+        PR_URL="https://github.com/${{ github.repository }}/pull/${{ github.event.issue.number }}"
+        echo "PR_URL=$PR_URL" >> $GITHUB_OUTPUT
+
+    - name: Obtain diff with the PR provided
+      if: steps.read_comment.outputs.other_pr
+      id: run_extension
+      env:
+        GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        gh extension install samugi/gh-compr --pin "1.0.0"
```

```suggestion
        gh extension install samugi/gh-compr --pin "3785a2d3270c52164fb1f7f63bd3c5df66bedead"
```

We usually pin by commit hash to ensure integrity of our supply chain.